pkgs/top-level/all-packages.nix

```diff
@@ -30384,6 +30384,10 @@ with pkgs;
     inherit (xfce) xfce4-dev-tools;
   };
 
+  lightdm_slick_greeter = callPackage ../applications/display-managers/lightdm-slick-greeter {
```

Comment:
Not sure if `lightdm-slick-greeter` will be a better attribute name? (FYI I am renaming lightdm_gtk_greeter to lightdm-gtk-greeter in #184454)

Reply:
Yeah, I was a bit confused on whether or not I should use underscores or dashes. I'll change it to that.